### ISSUE TITLE
Iphones touch

### ISSUE THREAD

**Issue by sinneren** [digitalBush/jquery.maskedinput#369]
Iphones 5,6.
When touch in the input, for ex. (999)-999-99-99, impossible to move cursor into text.

**Comment by rquadling** [comment]
I had an issue with the maxLength attribute. If that is present, and, depending upon the OS/browser, the mask was displayed as characters, attempting to enter another character (which the code would eventually treat as an overwrite), the insert of the character failed as the maxLength wouldn't allow for another character.

Removing maxLength and the mask worked fine.

Not sure if this is a similar issue for you.

**Comment by sinneren** [comment]
I don't think it similar :) I found this bug on any inputs in ios. Maybe it right in it, don't know, because i'm android user.
